app/models/invite_request.rb

```diff
@@ -14,6 +16,20 @@ def self.reset_order
     end
   end
 
+  # Borrow the blacklist cleaner but just strip out all the periods for all domains
+  def set_simplified_email
+    return if email.blank?
+    simplified = AdminBlacklistedEmail.canonical_email(email).split('@')
+    self.simplified_email = simplified.first.gsub(".", "") + "@#{simplified.last}"
```

Comment:
I would also do the + thing for all domains so people can't ask for two invites at the same address in the queue at one time. They can be patient and wait

Reply:
Under the circumstances, I'd rather err on the side of caution here. We're saving this as a different field so the original email is preserved and will be used to send the invite to. I can be more aggressive about stripping out the +s if you want.